vercel.json

```diff
@@ -54,6 +54,31 @@
         }
       ]
     },
+    {
+      "source": "/v/(.*)",
+      "headers": [
+        {
+          "key": "Access-Control-Allow-Origin",
+          "value": "*"
```

Comment:
I'm skeptical this will work too well, iirc "*" doesn't work as well as returning the request origin

Reply:
I think that is correct, although I cannot remember the cases where it might create an issue. Possibly with `localhost`. To further complicate things, the "rules" when disallowing credentials / for loading via `script` tag may be different than for fetch / XHR.